pinot-segment-local/src/main/java/org/apache/pinot/segment/local/function/GroovyFunctionEvaluator.java

```diff
@@ -96,6 +160,10 @@ public Object evaluate(GenericRow genericRow) {
     }
     try {
       return _script.run();
+    } catch (SecurityException ex) {
+      LOGGER.error("An insecure Groovy script execution caught: {}", _expression);
+      incrementSecurityViolationCounter();
+      throw ex;
```

Comment:
When we throw the exception out, no need to log error because caller is responsible for handling it. Same for other places